tck/src/main/java/org/eclipse/microprofile/lra/tck/TckLRATypeTests.java

```diff
@@ -215,6 +216,30 @@ public void neverWithEndRA() {
     public void neverWithoutEndLRA() {
         resourceRequest(NEVER_WITH_END_FALSE_PATH, false, 200, MethodLRACheck.NOT_PRESENT, false);
     }
+    
+    @Test
+    public void requiredWithClosedLRA() {
+        String closedLRA = resourceRequest(REQUIRED_PATH, false, 200, MethodLRACheck.NONE, false);
+     
+        // manually call REQUIRED LRA endpoint with the id of already finished LRA
+        Response response = tckSuiteTarget.path(TCK_LRA_TYPE_RESOURCE_PATH).path(REQUIRED_PATH).request()
+            .header(LRA.LRA_HTTP_CONTEXT_HEADER, closedLRA).get();
+
+        Assert.assertNotEquals("REQUIRED LRA endpoint called with inactive LRA context should " +
+            "start a new LRA", closedLRA, response.readEntity(String.class));
+    }
+    
+    @Test
+    public void mandatoryWithClosedLRA() {
+        String closedLRA = resourceRequest(REQUIRED_PATH, false, 200, MethodLRACheck.NONE, false);
```

Comment:
I think it might aid readability if this was:
> String lraId = resourceRequest(....
Of course, I might be wrong about what you are expecting here but I think it is an ID?

Reply:
will do, thank you

Reply:
thanks!